axlearn/common/flash_attention/layer.py

```diff
@@ -317,16 +321,18 @@ def default_output_dim_to_partition_spec(
 
 
 class FlashBlockSizeModifier(ConfigModifier):
-    """Modified the tpu_block_size config of FlashAttention."""
+    """Modifies the tpu_block_size or gpu_block_size config of FlashAttention."""
 
     @config_class
     class Config(ConfigModifier.Config):
         """Configures FlashBlockSizeModifier."""
 
-        tpu_block_size: Required[int] = REQUIRED
+        tpu_block_size: Optional[int] = 512
+        gpu_block_size: Optional[int] = 128
```

Comment:
```suggestion
        gpu_block_size: Optional[int] = None
```
As otherwise it incurs golden config changes across various configs. Since we've already set a default, we probably don't need to repeat it here.